### ISSUE TITLE
[Bug Report][2.3.16] Wrong aria-label of clickable VTextField append-icon

### ISSUE THREAD

**Issue by peterthomassen** [vuetifyjs/vuetify#12582]
### Environment
**Vuetify Version:** 2.3.16
**Vue Version:** 2.6.12
**Browsers:** Chrome 86.0.4240.75
**OS:** Linux x86_64

### Steps to reproduce
- Create a v-text-field with an `append-icon` attribute
- Add a `@click:append` attribute (turning the icon into a button)
- Inspect the button's `aria-label`

### Expected Behavior
The aria-label should be either
- reasonable (e.g. "field-appended action"), or
- configurable (e.g. `label:append="..."`, similar to `@click:append`)

### Actual Behavior
The aria-label is "append icon". This can be very misleading, especially if the appended icon is used for removing items from a list of fields.

### Reproduction Link
<a href="https://codepen.io/peterthomassen/pen/GRqwoeQ" target="_blank">https://codepen.io/peterthomassen/pen/GRqwoeQ</a>

### Other comments
I believe the cause for the observed behavior is here: https://github.com/vuetifyjs/vuetify/blob/78b0ae2bd2d8b843417250f0cd3bcecf99604112/packages/vuetify/src/components/VInput/VInput.ts#L179

**Comment by KaelWD** [comment]
Use the `append` slot to provide your own icon with the desired attributes.

**Comment by randomstuff** [comment]
> Use the append slot to provide your own icon with the desired attributes.

This fixes the issue but is not very satisfying. 

For reference, this gives something like:

~~~vue
<v-text-field
  :type="showPassword ? 'text' : 'password'"
  name="password"
  autocomplete="current-password"
  label="Password"
  prepend-icon="mdi-lock"
  v-model="password">
  <template #append>
    <v-btn icon @click="showPassword=!showPassword"
        :aria-label="showPassword ? 'Hide password' : 'Show password'">
      <v-icon>{{showPassword ? 'mdi-eye-off' : 'mdi-eye'}}</v-icon>
    </v-btn>
  </template>
</v-text-field>
~~~

**Comment by KaelWD** [comment]
I don't think it's gonna get much cleaner than this. I don't like `label:append` because it's too specific, and `append-props` would just be a slot with extra steps. 

```html
<v-text-field
  v-model="password"
  :type="showPassword ? 'text' : 'password'"
  name="password"
  autocomplete="current-password"
  label="Password"
  prepend-icon="mdi-lock"
>
  <template #append>
    <v-btn 
      :icon="showPassword ? 'mdi-eye-off' : 'mdi-eye'" 
      :aria-label="showPassword ? 'Hide password' : 'Show password'"
      tabindex="-1"
      @click="showPassword =! showPassword"
    />
  </template>
</v-text-field>
```
